### ISSUE TITLE
PyQt version import error prevents application from launching

### ISSUE THREAD

**Issue by marek-lach** [vkbo/novelWriter#1324]
Sorry for the inconvenience, but is there any idea what this output could possibly mean?

[https://pastebin.com/H8nXudaX](https://pastebin.com/H8nXudaX)

Thanks for looking into it.

**Comment by vkbo** [comment]
Pasting the backtrace here so I don't have to go to a third party service:

```
Traceback (most recent call last):
  File "/usr/bin/novelwriter", line 33, in <module>
    sys.exit(load_entry_point('novelWriter==2.0.3', 'gui_scripts', 'novelwriter')())
  File "/usr/bin/novelwriter", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 171, in load
    module = import_module(match.group('module'))
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/usr/lib/python3.10/site-packages/novelwriter/__init__.py", line 33, in <module>
    from novelwriter.config import Config
  File "/usr/lib/python3.10/site-packages/novelwriter/config.py", line 33, in <module>
    from PyQt5.Qt import PYQT_VERSION_STR
ImportError: cannot import name 'PYQT_VERSION_STR' from 'PyQt5.Qt' (/usr/lib/python3.10/site-packages/PyQt5/Qt.abi3.so)
```

**Comment by vkbo** [comment]
The `PYQT_VERSION_STR` is something PyQt5 adds. Maybe they've made a change?

What PyQt5 version is actually installed, and from what source?

**Comment by vkbo** [comment]
It is still in their docs, but importable from QtCore: https://www.riverbankcomputing.com/static/Docs/PyQt5/api/qtcore/qtcore-module.html?highlight=pyqt_version_str##PYQT_VERSION_STR

I see that novelWriter imports it from Qt directly, so perhaps that was just an import bleed through that has been changed. If so, this is a bug in novelWriter and needs to be fixed.

**Comment by vkbo** [comment]
I've made a fix, and I'll see if I can complete some other fixes tomorrow and do a 2.0.4 release.

**Comment by vkbo** [comment]
Aha! Mystery solved. This is a change in PyQt5 version 5.15.8 which was released a few hours ago.
https://www.riverbankcomputing.com/news/PyQt_v5.15.8_Released

All my CI jobs on GitHub started to fail now as they pull in the latest version. I have 5.15.2 on my system.

**Comment by vkbo** [comment]
Fixed and released 2.0.4.
